packages/nextjs-mf/src/internal.ts

```diff
@@ -100,11 +100,11 @@ export const generateRemoteTemplate = (url: string, global: any) => `new Promise
     var url = new URL(${JSON.stringify(url)});
     url.searchParams.set('t', Date.now());
     var __webpack_error__ = new Error();
-    if (typeof ${global} !== 'undefined') return resolve();
+    if (typeof ${global} !== 'undefined') return resolve(${global});
```

Comment:
Not really required since remotes register to globals. But resolving for better consistency